shared/types/index.ts

```diff
@@ -20,6 +20,10 @@ export interface MediaProps extends React.HTMLAttributes<HTMLDivElement> {
     license?: string
     link?: string
   }
+  srcset?: string
+  image_srcset?: string
```

Comment:
What's the difference between `srcset` and `image_srcset`?

Reply:
I simplified this